test/manual/definitions/apm/java/linux/tomcat/systemd/deb10-sysd-catalina-sh.json

```diff
@@ -19,7 +19,7 @@
   "services": [
     {
       "id": "java1",
-      "source_repository": "https://github.com/newrelic/open-install-library.git",
+      "local_source_path": "open-install-library/",
```

Comment:
@mbruzina you might want to undo this file change

Reply:
I'll push that fix to main (maven version bump)